docs/cell_census_schema_0.0.1.md

```diff
@@ -0,0 +1,853 @@
+# CELLxGENE Cell Census Schema 
+
+**Status**: Draft.
+
+**Version**: 0.0.1.
+
+**Last edited**: Dec, 2022.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14), [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+## Cell Census overview
+
+The Cell Census is a versioned container for the single-cell data hosted at [CELLxGENE Discover](https://cellxgene.cziscience.com/). The Cell Census utilizes [SOMA](https://github.com/single-cell-data/TileDB-SOMA) powered by [TileDB](https://tiledb.com/products/tiledb-embedded) for storing, accessing, and efficiently filtering data.
```

Comment:
implies that there is equivalence in the data present at both locations.  Cell Census is initially going to be a subset of the Portal data.